cmd/siftest/readme.md

```diff
@@ -0,0 +1,137 @@
+# Siftest User Guide
+
+## About
+
+This can be used after running transactions to verify expected vs real changes.
+
+## Installation
+
+```shell
+git clone git@github.com:Sifchain/sifnode.git
+cd sifnode
+make install
+```
+
+## Verify Add Liquidity
+
+1. Execute add liquidity transaction
+2. Verify by passing in the following arguments
+   1. --height [height of transaction]
+   2. --from [address of transactor]
+   3. --external-asset [external asset of pool]
+   4. --nativeAmount [native amount requested to add]
+   5. --externalAmount [external amount requested to add]
+   6. --node [node to connect to]
+```shell
+siftest verify add --from sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd --height=43516 --external-asset=ceth --nativeAmount=96176925423929435353999282 --externalAmount=488436982990 --node tcp://localhost:26657
+```
+
+Output
+```shell
+verifying add...
+
+Wallet native balance before 499999807448801156767565321374116
+Wallet external balance before 499999999999999999999022148656694
+
+Wallet native balance after 499999711271875632838129967374834 
+Wallet external balance after 499999999999999999998533711673704 
+
+Wallet native diff -96176925523929435353999282 (expected: -96176925423929435353999282 unexpected: -100000000000000000)
+Wallet external diff -488436982990 (expected: -488436982990 unexpected: 0)
+
+LP units before 192542049745763466715763665 
+LP units after 288716849962488234851636499 
+LP units diff 96174800216724768135872834 (expected: 96174800216724768135872834)
+
+Pool share before 1.000000000000000000
+Pool share after 1.000000000000000000
+```
+
+## Verify Remove Liquidity
+
+1. Execute remove liquidity transaction
+2. Verify by passing in the following parameters:
+   1. --height [height of transaction]
+   2. --from [address of transactor]
+   3. --external-asset [external asset of pool]
+   4. --units [units requested for removal]
+   5. --node [node to connect to]
+Command
+```shell
+siftest verify remove --from sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd --units 1000000000000000000 --height=33068 --external-asset=ceth --node tcp://localhost:26657
+```
+
+Output
+```shell
+verifying removal...
+
+Wallet native balance before 499999807448701559072997868307527
+Wallet external balance before 499999999999999999999022148651615
+
+Wallet native balance after 499999807448702459100223367291740 
+Wallet external balance after 499999999999999999999022148656694 
+
+Wallet native diff 900027225498984213 (expected: 1000032419169645384 unexpected: -100005193670661171)
```

Comment:
do we want to show in the examples a failure case (think this is what `unexpected` states)?